docs/guides/installation_uv.md

```diff
@@ -0,0 +1,260 @@
+
+# Running Marimo with `uv`
+
+You can run a Marimo notebook using the package manager `uv` through one of the following methods:
+
+1. **Within a project environment**  
+2. **Temporary installation (cached)**  
```

Comment:
maybe a "ephemeral virtual enviroment"

Reply:
I agree, 
Temporary installation (cached) is not the best choice.

I think "Ephemeral" is not widely recognized, especially by non-native English speakers.
How about:
Short-Lived Environment

But I would also love to find a better word for 
"self-contained scripts that run with temporary enviornments"